snapcraft/internal/repo/_platform.py

```diff
@@ -26,6 +26,7 @@
     '"elementary"',
     'debian',
     'neon',
+    'LinuxMint',
```

Comment:
1. I has run script:
'(echo "import platform" ; echo "print (platform.linux_distribution()[0])") | strace python3'
2. View logs:
'
......
open("/etc/lsb-release", O_RDONLY|O_CLOEXEC) = 3
fstat(3, {st_mode=S_IFREG|0644, st_size=105, ...}) = 0
ioctl(3, TCGETS, 0x7fffcf805070)        = -1 ENOTTY (Inappropriate ioctl for device)
lseek(3, 0, SEEK_CUR)                   = 0
ioctl(3, TCGETS, 0x7fffcf805010)        = -1 ENOTTY (Inappropriate ioctl for device)
lseek(3, 0, SEEK_CUR)                   = 0
read(3, "DISTRIB_ID=LinuxMint\nDISTRIB_REL"..., 8192) = 105
read(3, "", 8192)                       = 0
close(3)                                = 0
write(1, "LinuxMint\n", 10LinuxMint
......
'
3. And dump file:
cat /etc/lsb-release
DISTRIB_ID=LinuxMint
DISTRIB_RELEASE=18
DISTRIB_CODENAME=sarah
DISTRIB_DESCRIPTION="Linux Mint 18 Sarah"